### ISSUE TITLE
Export Signer Class

### ISSUE THREAD

**Issue by jacob-tucker** [doublejumptokyo/fcl-kms-authorizer#17]
Should the Signer class be exported in src/index.ts? I believe only the Authorizer is exported, but I would like to use the Signer class as well for its `sign` function without having to have the entire authorization function. I just want to sign a message with my KMS keys.

**Comment by avcdsld** [comment]
@jacob-tucker Thank you for the feature request suggestion.
I recognize that this is a necessary feature and will fix it immediately.

**Comment by avcdsld** [comment]
@jacob-tucker Your requested feature has been incorporated and released in v0.0.13. Thank you!

You can create a user signature as follows!

```js
const signer = authorizer.getSigner();

const signature = await signer.signUserMessage("test message");

// or just sign without userTag
const signatureWithoutUserTag = await signer.sign("test message");
```

or 

```js
import { Signer } from "fcl-kms-authorizer";

const signer = new Signer(
    {
      credentials: fromEnv(),
      region,
    },
    keyIds
  );
const signature = await signer.signUserMessage("test message");

// or just sign without userTag
const signatureWithoutUserTag = await signer.sign("test message");
```

I close this ticket.
